### ISSUE TITLE
Can I save to Faiss?

### ISSUE THREAD

**Issue by duckheada** [OML-Team/open-metric-learning#445]

**Comment by AlekseySh** [comment]
Hey, @duckheada 
Thank you for using OML.

Are you talking about inference time?
If so, you can take an example from [Readme](https://github.com/OML-Team/open-metric-learning#examples) named `Using a trained model for retrieval`.

You simply need to replace `sklearn.NearestNeighbors` by `Faiss`
